benchmarking.ipynb

```diff
@@ -0,0 +1,428 @@
+{
```

Comment:
<p> <code>/transformers4rec/TF4Rec/models/</code>  --&gt; when and how these models were exported to that folder?&nbsp; can you add some explanations?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Transformers4Rec/pull/615/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>absolutely! added a notebook with steps for training and exporting models</p>